conduct.md

```diff
@@ -96,9 +96,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should not update documentation or code without consulting the group about it.
```

Comment:
Is it possible that this statement is too restrictive? Could it be better to say that students should make sure to change documentation or code in ways that are consistent with the decisions made by the team?